bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2021A.xml

```diff
@@ -0,0 +1,175 @@
+    </type>
+
+    <type name="PoorWeather">
+        <dir>/home/software/proposals/poor_weather/proposals/2021A</dir>
+        <to>pitpw@gemini.edu</to>
+        <template>pw</template>
+    </type>
+
+    <type name="DemoScience">
+        <dir>/home/software/proposals/demo_science/proposals/2021A</dir>
+        <to>pitds@gemini.edu</to>
+    </type>
+
+    <type name="SystemVerification">
+        <dir>/home/software/proposals/system_verification/proposals/2021A</dir>
+        <to>pitsv@gemini.edu</to>
+    </type>
+
+    <type name="GuaranteedTime">
+        <dir>/home/software/proposals/guaranteed_time/proposals/2021A</dir>
+        <to>pitgt@gemini.edu</to>
+    </type>
+
+    <type name="LargeProgram">
+        <dir>/home/software/proposals/large_program/proposals/2021A</dir>
+        <to>pitlp@gemini.edu</to>
+    </type>
+
+    <type name="SIP">
+        <dir>/home/software/proposals/subaru_intensive_program/proposals/2021A</dir>
+        <to>pitlp@gemini.edu</to>
+    </type>
+
+    <type name="ar">
+        <dir>/home/software/proposals/ar/proposals/2021A</dir>
+        <to>pit.ar@gemini.edu</to>
+    </type>
+
+    <type name="au">
+        <dir>/home/software/proposals/au/proposals/2021A</dir>
+        <to>pit.au@gemini.edu</to>
+        <template>au</template>
+    </type>
+
+    <type name="br">
+        <dir>/home/software/proposals/br/proposals/2021A</dir>
+        <to>pit.br@gemini.edu</to>
+    </type>
+
+    <type name="ca">
+        <dir>/home/software/proposals/ca/proposals/2021A</dir>
+        <to>pit.ca@gemini.edu</to>
+    </type>
+
+    <type name="cfh">
+        <dir>/home/software/proposals/cfh/proposals/2021A</dir>
+        <to>pit.cfh@gemini.edu</to>
+    </type>
+
+    <type name="cl">
+        <dir>/home/software/proposals/cl/proposals/2021A</dir>
+        <to>pit.cl@gemini.edu</to>
+        <template>cl</template>
+    </type>
+
+    <type name="kr">
+        <dir>/home/software/proposals/kr/proposals/2021A</dir>
+        <to>pit.kr@gemini.edu</to>
+    </type>
+
+    <type name="uh">
+        <dir>/home/software/proposals/uh/proposals/2021A</dir>
+        <to>pit.uh@gemini.edu</to>
+    </type>
+
+    <type name="us">
+        <dir>/home/software/proposals/us/proposals/2021A</dir>
+        <to>pit.us@gemini.edu</to>
+        <template>us</template>
+    </type>
+
+    <type name="subaru">
+        <dir>/home/software/proposals/subaru/proposals/2021A</dir>
+        <to>pit.subaru@gemini.edu</to>
+    </type>
+
+    <filetranslation>
+        <translation>
+            <from>ft</from>
+            <to>FT</to>
+        </translation>
+        <translation>
+            <from>System Verification</from>
+            <to>SV</to>
+        </translation>
+        <translation>
+            <from>Poor Weather</from>
+            <to>PW</to>
+        </translation>
+        <translation>
+            <from>Directors Time</from>
+            <to>DT</to>
+        </translation>
+        <translation>
```

Comment:
We may need a translation from `Guaranteed Time` to `GT` this is used to name the saved proposal files